RELEASE.md

```diff
@@ -1,9 +1,42 @@
-# Release Process
+# Release Guide
 
-The Kubernetes Template Project is released on an as-needed basis. The process is as follows:
+## Semi-automatic
 
-1. An issue is proposing a new release with a changelog since the last release
-1. All [OWNERS](OWNERS) must LGTM this release
-1. An OWNER runs `git tag -s $VERSION` and inserts the changelog and pushes the tag with `git push $VERSION`
-1. The release issue is closed
-1. An announcement email is sent to `kubernetes-dev@googlegroups.com` with the subject `[ANNOUNCE] kubernetes-template-project $VERSION is released`
+1. Make sure your repo is clean by git's standards
+2. Create a release branch `git checkout -b release-1.19` (not required for patch releases)
+3. Push the release branch to the kube-scheduler-simulator repo and ensure branch protection is enabled (not required for patch releases)
+4. Tag the repository from the `master` branch (from the `release-1.19` branch for a patch release) and push the tag `VERSION=v0.19.0 git tag -m $VERSION $VERSION; git push origin $VERSION`
+5. Publish a draft release using the tag you just created
+6. Perform the [image promotion process](https://github.com/kubernetes/k8s.io/tree/main/k8s.gcr.io#image-promoter)
+7. Publish release
+8. Email `kubernetes-sig-scheduling@googlegroups.com` to announce the release
+
+## Manual
+
+1. Make sure your repo is clean by git's standards
+2. Create a release branch `git checkout -b release-1.19` (not required for patch releases)
+3. Push the release branch to the kube-scheduler-simulator repo and ensure branch protection is enabled (not required for patch releases)
+4. Tag the repository from the `master` branch (from the `release-1.19` branch for a patch release) and push the tag `VERSION=v0.19.0 git tag -m $VERSION $VERSION; git push origin $VERSION`
+5. Checkout the tag you just created and make sure your repo is clean by git's standards `git checkout $VERSION`
+6. Build and push the container image to the staging registry `RELEASE_VERSION=$VERSION make push-release-image`
```

Comment:
Who has permission to do this?

Why not only document the "semi-automatic" way?

Reply:
The takeaway of this section is that some forked repo can follow this to release and push images to their own image registry, but maybe not that useful. So I'd like to either remove the whole section or reword it to fit the purpose of forked users.

Reply:
OK. I'll delete it.